packages/notifi-react-card/lib/components/subscription/defaults.css

```diff
@@ -6,7 +6,7 @@
   --notifi-secondary-font-color: rgb(93, 93, 93);
   --notifi-input-background: rgba(255, 255, 255, 0.8);
   --notifi-intercom-input-background: rgb(245, 246, 251, 1);
-  --notifi-placeholder-color: rgba(224, 224, 224, 0.5);
+  --notifi-placeholder-color: rgb(33 33 33 / 50%);
```

Comment:
woah, what's this syntax?

Reply:
ohh it's another way for opacity. the current light one was a little too hard to read